src/sql/workbench/api/common/extHostModelView.ts

```diff
@@ -79,7 +79,12 @@ class ModelBuilderImpl implements azdata.ModelBuilder {
 		return container;
 	}
 
+	private cardDeprecationMessagePrinted = false;
 	card(): azdata.ComponentBuilder<azdata.CardComponent> {
+		if (!this.cardDeprecationMessagePrinted) {
+			console.warn(`Extension '${this._extension.identifier.value}' is using card component which has been replaced by radioCardGroup. the card component will be removed in a future release.`);
```

Comment:
Use ILogService instead of logging directly with console, and put the [@deprecated](https://github.com/microsoft/azuredatastudio/blob/master/src/vs/vscode.d.ts#L1157) doc tag in azdata.d.ts

Reply:
leaving it as console for now.